--- a/tc/q_taprio.c
+++ b/tc/q_taprio.c
@@ -99,6 +99,10 @@ static const char *entry_cmd_to_str(__u8
 	switch (cmd) {
 	case TC_TAPRIO_CMD_SET_GATES:
 		return "S";
+	case TC_TAPRIO_CMD_SET_AND_HOLD:
+		return "SH";
+	case TC_TAPRIO_CMD_SET_AND_RELEASE:
+		return "SR";
 	default:
 		return "Invalid";
 	}
@@ -109,6 +113,12 @@ static int str_to_entry_cmd(const char *
 	if (strcmp(str, "S") == 0)
 		return TC_TAPRIO_CMD_SET_GATES;
 
+	if (strcmp(str, "SH") == 0)
+		return TC_TAPRIO_CMD_SET_AND_HOLD;
+
+	if (strcmp(str, "SR") == 0)
+		return TC_TAPRIO_CMD_SET_AND_RELEASE;
+
 	return -1;
 }
 
