From e2aa6182cb4d08a188adc3ab69c0806ac82f945b Mon Sep 17 00:00:00 2001
From: Biao Huang <biao.huang@mediatek.com>
Date: Mon, 20 Jun 2022 14:48:59 +0800
Subject: [PATCH 2/2] modify gatemask to match tc order on MediaTek platform

---
 dump-classifier.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dump-classifier.c b/dump-classifier.c
index 5813367..0f8e050 100644
--- a/dump-classifier.c
+++ b/dump-classifier.c
@@ -476,7 +476,7 @@ static int classify_frames(pcap_t *handle, const struct tc_filter *tc_filters,
 		else
 			name = "BE";
 
-		if (entry->gatemask & (1 << tc))
+		if (entry->gatemask & (1 << (3 - tc)))
 			ontime = "ontime";
 		else
 			ontime = "late";
-- 
2.18.0

